README.md

```diff
@@ -1 +1,177 @@
+
+The website content is fictional. It was created using HTML, CSS and JavaScript to meet the requirements defined within the scope of the Microverse Capstone project. In particular, I would like to point out that I built this website completely from scratch.
+
+
+![web](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/images/conference_page.png)
+
+**Microverse HTML, CSS, JS Capstone Projec** is a fictional website describing a conference on the evolution of digital in our continent and more precisely in our region. It is built using HTML, CSS, and JavaScript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JS</a></li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Javascript Capstone**
+- **React.js Capstone**
+- **ROR Capstone**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [noel-capstone](https://noelfoka.github.io/first-capstone-project/)
+
+## Presentation of Project
+- [Loom Video Link](https://www.loom.com/share/e97d1587121a4dc9a90db8145b204eb0)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ npm install
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd first-capstone-project
+  git clone git@github.com:noelfoka/first-capstone-project.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Noel Foka**
+
+- GitHub: [@noelfoka](https://github.com/noelfoka)
+- Twitter: [@noelnomgne](https://twitter.com/noelnomgne)
+- LinkedIn: [noel-foka](https://www.linkedin.com/in/no%C3%ABl-nomgne-foka-063013231/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Javascript Capstone project**
+- [ ] **React JS capstone project**
+- [ ] **ROR Capstone project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me the ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
```

Comment:
Please add appropriate credit to Cindy Shin so that I can approve your project

For Example:
`This template was inspired by [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)`

Reply:
It's ok